test/ig/IGExporter.test.ts

```diff
@@ -171,6 +171,20 @@ describe('IGExporter', () => {
             uri: 'http://hl7.org/fhir/uv/vhdir/ImplementationGuide/hl7.core.uv.vhdir',
             packageId: 'hl7.fhir.uv.vhdir',
             version: 'current'
+          },
+          // test-wonky-id-1 tests that we are correctly turning - in packageId to - in id
```

Comment:
This is the smallest comment I have ever made, but I think the second `-` in this comment should be `_`

Reply:
I can't believe I've created such a flawed PR. Shame on me, shame!